tests/TestCase/Routing/RouteBuilderTest.php

```diff
@@ -569,7 +569,7 @@ public function testResourcesPathOption(): void
     public function testResourcesPrefix(): void
     {
         $routes = new RouteBuilder($this->collection, '/api');
-        $routes->resources('Articles', ['prefix' => 'Rest']);
+        $routes->resources('Articles', null, ['prefix' => 'Rest']);
```

Comment:
```suggestion
        $routes->resources('Articles', options: ['prefix' => 'Rest']);
```

Should we have a test for the named parameter form? If we're going to include it in the docs we should have a test on it.